app/models/question.rb

```diff
@@ -18,8 +18,10 @@
 class Question < ApplicationRecord
   belongs_to :documentation
   validates :body, :answers, :correct_answers, :explanation,
-             :documentation_id, :answer_counter, :positive_rates,
-             :negative_rates, presence: true
+            :documentation_id, :answer_counter, :positive_rates,
+            :negative_rates, presence: true
+
+  before_save :convert_correct_answers
 
   def correct_count
     QuestionService.new(self).find_correct_count
```

Comment:
Calling a service from a model is quite unusual.
The controller would probably be a better place for that, since its goal is the be the "glue" between every pieces.
The model's only responsibility is to care for itself and its attributes.

That is said: let's let it here for now, and we'll be able to move it somewhere else later, if we feel uncomfortable using it. ;)